Gandalan.IDAS.WebApi.Client/DTOs/API/ApiVersionDTO.cs

```diff
@@ -1,17 +1,46 @@
-﻿using System.Reflection;
+﻿using System;
+using System.Reflection;
 
 namespace Gandalan.IDAS.WebApi.Client.DTOs.API
 {
     public class ApiVersionDTO
     {
         public string Version { get; set; }
         public string Environment { get; set; }
-        
+        public string BuildDate { get; set; }
+
         public static ApiVersionDTO FromAssembly(Assembly assembly)
         {
-            var version = assembly.GetCustomAttribute<AssemblyInformationalVersionAttribute>()?.InformationalVersion ?? $"No informational version found for {assembly.FullName}";
+            var version = assembly.GetCustomAttribute<AssemblyInformationalVersionAttribute>()?.InformationalVersion ?? $"Keine Version gefunden für: {assembly.FullName}";
             var env = System.Environment.GetEnvironmentVariable("GDL_ENVIRONMENT") ?? "Development";
-            return new ApiVersionDTO() { Version = version, Environment = env };
+            var buildDate = ExtractBuildDateFromAssembly(assembly) ?? "Build-Datum nicht verfügbar";
+
+            return new ApiVersionDTO() { Version = version, Environment = env, BuildDate = buildDate };
```

Comment:
```suggestion
            return new ApiVersionDTO() {
                Version = version,
                Environment = env,
                BuildDate = buildDate,
            };
```

Reply:
@gdl-reif thought it would be best to use both: ReleaseTime and BuildTime. 
Is that okay for you @Saibamen ? This would mean you need to extend IBOS1/2/legacy api-version controllers.

Reply:
Or just `ReleaseTime = System.Environment.GetEnvironmentVariable("RELEASE_DEPLOYMENT_STARTTIME") ?? ExtractBuildTimeFromAssembly(assembly),`, without `BuildTime` property

Reply:
Hm, wouldn't that be confusing?

Reply:
@Saibamen In any case, we need BuildTime AND ReleaseTime if available. ReleaseTime is usually not that important and can be very different

Reply:
OK, but I don't know what I need to display for `BuildTime` for IBOSes microservices - our `ReleaseTime` comes from DevOps Release Pipeline start time and is set at the end of pipeline (so if we have failed building Docker image, `ReleaseTime` will not be updated).

1. Do nothing (return model for I1/I2 MS will not have `BuildTime`)
2. Extend our IBOS version model with `BuildTime`, but it will be always `null`
3. Extend and put this same value as for `ReleaseTime`?

Reply:
Sounds like the `ReleaseTime `is more like the `BuildTime`? Maybe we really just _omit_ the `ReleaseTime` in the `ApiVersionDTO`. For IBOS1/2-microservices you could go with the third proposal: Extend and put same value as for `ReleaseTime`.

Reply:
`ReleaseTime` for me is the time the service went into production, based on any build that has a `BuildTime`. In terms of testing, these two dates can widely differ. One is the build pipeline run time, the other the time the release pipeline started. Both values can and should be filled, so I think we must adapt the build pipeline for the microservices to include the build pipeline start time.

Reply:
@Saibamen as we have working pipelines which set the `BuildTime` field, I think this can be provided for